### ISSUE TITLE
Destructuring defaults

### ISSUE THREAD

**Issue by hemanth** [6to5/6to5#403]
```
let myMeth = ({ans=420}) => {console.log('Answer: ',ans)}

myMeth({});
```

Results in:

```
SyntaxError: repl: Unexpected token (1:18)
> 1  | let myMeth = ({ans=420}) => {console.log('Answer: ',ans)}
     |                  ^
  2  | 
  3  | myMeth({});
  4  | 
    at raise (https://6to5.org/scripts.js:1:314788)
    at unexpected (https://6to5.org/scripts.js:1:341818)
    at expect (https://6to5.org/scripts.js:1:341732)
    at parseObj (https://6to5.org/scripts.js:1:361464)
    at parseExprAtom (https://6to5.org/scripts.js:1:359839)
    at parseExprSubscripts (https://6to5.org/scripts.js:1:355723)
    at parseMaybeUnary (https://6to5.org/scripts.js:1:355389)
    at parseExprOps (https://6to5.org/scripts.js:1:354292)
    at parseMaybeConditional (https://6to5.org/scripts.js:1:353676)
    at parseMaybeAssign (https://6to5.org/scripts.js:1:353302)
```

**Comment by sebmck** [comment]
See #230

**Comment by hemanth** [comment]
@sebmck Thanks, searched before filling, but did not find.

**Comment by sebmck** [comment]
Currently available as of 2.9.0

**Comment by hemanth** [comment]
:+1:
